unison-cli/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -3080,9 +3080,9 @@ doSlurpAdds slurp uf = Branch.batchUpdates (typeActions <> termActions)
     doTerm :: v -> (Path, Branch0 m -> Branch0 m)
     doTerm v = case toList (Names.termsNamed names (Name.unsafeFromVar v)) of
       [] -> errorMissingVar v
-      [r] -> case Path.splitFromName (Name.unsafeFromVar v) of
-        Nothing -> errorEmptyVar
-        Just split -> BranchUtil.makeAddTermName split r (md v)
+      [r] ->
+        let split = Path.splitFromName (Name.unsafeFromVar v)
```

Comment:
`splitFromName` was returning a Maybe even though it could never fail since names are non-empty; so all of these changes just remove the redundant 'maybe' handling.